package.json

```diff
@@ -87,6 +87,8 @@
     "url-loader": "^0.5.7",
     "webpack": "1.12.14",
     "webpack-dev-middleware": "^1.2.0",
-    "webpack-dev-server": "^1.14.1"
+    "webpack-dev-server": "^1.14.1",
+    "react-virtualized": "^7.3.1",
+    "react-addons-shallow-compare": "^15.1.0"
```

Comment:
There's no need to keep it under dependencies just in case someone wants to switch back. It's better to remove it from `package.json` and who really wants to use fixed-data-table can easly install it manually.